wsagent/che-datasource-api/src/main/java/org/eclipse/che/datasource/api/ssl/KeyStoreObject.java

```diff
@@ -0,0 +1,184 @@
+package org.eclipse.che.datasource.api.ssl;
+
+import com.google.gson.reflect.TypeToken;
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import org.apache.commons.fileupload.FileItem;
+import org.apache.commons.codec.binary.Base64;
+import org.eclipse.che.api.core.ApiException;
+import org.eclipse.che.api.core.rest.HttpJsonHelper;
+import org.eclipse.che.commons.env.EnvironmentContext;
+import org.eclipse.che.commons.json.JsonHelper;
+import org.eclipse.che.commons.json.JsonParseException;
+import org.eclipse.che.datasource.shared.ssl.SslKeyStoreEntry;
+import org.eclipse.che.dto.server.DtoFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.*;
```

Comment:
In Che we're trying to avoid such style of importing, it would be very nice of you if you replace that with direct imports.